### PR TITLE
Move backend lint config to cli config

### PR DESCRIPTION
**Pull request by freben**
#### :heavy_check_mark: Checklist

- [ ] All tests are passing `yarn test`
- [ ] Screenshots attached (for UI changes)
- [ ] Relevant documentation updated
- [ ] Prettier run on changed files
- [ ] Tests added for new functionality
- [ ] Regression tests added for bug fixes

### CODE REVIEW

File: packages/backend/.eslintrc.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 module.exports = {
-  rules: {
-    'no-console': 0, // Permitted in console programs
-    'new-cap': ['error', { capIsNew: false }], // Because Express constructs things e.g. like 'const r = express.Router()'
-  },
+  extends: [
+    require.resolve('@backstage/cli/config/eslint.backend'),
+    '@spotify/eslint-config-oss',
```
Comment (Rugvip) [inline packages/backend/.eslintrc.js:4]: eslint-config-oss should be picked up by the root config afaik

Comment (freben) [reply packages/backend/.eslintrc.js:4]: Hm but the root also includes `@backstage/cli/config/eslint` which i DON'T want. I was hoping that it wouldn't 🤔

Comment (Rugvip) [reply packages/backend/.eslintrc.js:4]: Right so let's keep it and make sure the parent config isn't picked up. Afaik you need to set `root: true` either here or in the shared config.

**Review by Rugvip — APPROVED** [review]
👍

**Pull request merged**